src/Features/Core/Portable/Diagnostics/IDiagnosticServiceExtensions.cs

```diff
@@ -32,5 +32,8 @@ public static ImmutableArray<DiagnosticData> GetDiagnostics(this IDiagnosticServ
 
             return result.ToImmutable();
         }
+
+        public static ImmutableArray<DiagnosticBucket> GetDiagnosticBuckets(this IDiagnosticService service, Workspace workspace, DocumentId documentId, CancellationToken cancellationToken)
```

Comment:
I may have missed it, is this called in the PR somewhere?

Reply:
it is not.  i added it initially for some code, but ended up not using it.  nice catch :)